packages/engine/lib/error/Cargo.toml

```diff
@@ -19,7 +19,7 @@ rustc_version = "0.2.3"
 [features]
 default = ["std", "hooks"]
 std = []
-hooks = ["dep:once_cell"]
+hooks = ["dep:once_cell", "std"]
```

Comment:
`once_cell::sync` (The thing we are using for setting the hooks) requires `std`, thus implying it here as well (compile error otherwise). Added this to the crate level docs as well.